ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/CCodeGeneratorAmazonS3InitRequests_Test.java

```diff
@@ -100,10 +110,12 @@ public void testGenerateInitAmazonS3PutBucketRequest_NoPayload() throws IOExcept
         final TestFileUtilsImpl fileUtils = new TestFileUtilsImpl();
         final Ds3SpecParser parser = new Ds3SpecParserImpl();
         final Ds3ApiSpec spec = parser.getSpec(CCodeGenerator_Test.class.getResourceAsStream(inputSpecFile));
+        final Ds3DocSpecParser docSpecParser = new Ds3DocSpecParserImpl(new NameMapper());
+        final Ds3DocSpec docSpec = docSpecParser.getDocSpec(CCodeGenerator_Test.class.getResourceAsStream("/input/testCommandDocumentation.json"));
```

Comment:
This doc-spec is created in multiple steps, adding duplicate code. Either move to separate function, i.e. createTestDocSpec, or make private static var in test class.